posts/how-to-commit-code-to-a-git-repository-with-visual-studio-code.md

```diff
@@ -0,0 +1,172 @@
+
+```
+Update Readme and Getting Started docs
+```
+
+We’ll use the **Message** input box at the top of the **Source Control** panel to enter this commit message.
+
+<figure className="small">
+    <img src="https://storage.googleapis.com/sourcegraph-assets/learn/tutorial-images/vs-code-source-control-staged-changes-with-message.png" alt="Writing a commit message in the Message input box." />
+    <figcaption>Writing a commit message in the <strong>Message</strong> input box.</figcaption>
+</figure>
+
+A common convention in Git is to limit the length of commit messages to a maximum of 50 characters. VS Code helps us follow this convention by showing a warning message if we exceed that length.
+
+We’ll use the **Commit** button located at the very top of the **Source Control** panel to create the commit. This button appears as a checkmark icon, visible in the following screenshot:
+
+<figure className="small">
+    <img src="https://storage.googleapis.com/sourcegraph-assets/learn/tutorial-images/vs-code-source-control-commit-button.png" alt="The Commit button at the top of the Source Control panel." />
+    <figcaption>The <strong>Commit</strong> button at the top of the <strong>Source Control</strong> panel.</figcaption>
+</figure>
+
+When we click the **Commit** button, a new commit is created containing the staged changes. Those changes will disappear from the **Source Control** panel, and the commit message input box will be cleared. Any unstaged changes will remain unstaged and available for inclusion in future commits.
+
+## Step 6 — Checking the commit
+
+To check that the commit was successful, we can view the commit log and confirm that our commit is present in the repository. Although VS Code doesn’t have a built-in way to display the commit log, we can display it by running a Git command in the terminal: `git log`.
+
+Open the integrated terminal in VS Code by clicking on the **View** menu and clicking on **Terminal**. This should open a **Terminal** panel in the lower half of the window. The integrated terminal can also be opened with the keyboard shortcut: Control-<code>`</code> (Control and backtick).
+
+To display the commit log in the terminal, run the `git log` command. By default, the command will display a scrollable list of all previous commits. In this case, we’re only interested in checking the latest changes we made, so we can provide the `-1` flag to the command to limit its output to one commit. Here’s what we’ll run in the terminal:
+
+```sh
+git log -1
+```
+
+Here’s the result of the command:
+
+```
+commit 5388e04d38e5de13e3968f8d5e1932a9b41e5e53 (HEAD -> main)
+Author: Marek <marek@sourcegraph.com>
+Date:   Tue Aug 31 18:47:11 2021 -0400
+
+    Update Readme and Getting Started docs
+```
+
+In the above output of `git log -1`, we see the commit ID and the branch name (`main`) on the first line. The output also includes the author and date, followed by the commit message.
```

Comment:
Avoid "see":

"In the above output of `git log -1`, we receive the output of the commit ID and the branch name (`main`) on the first line."